### ISSUE TITLE
Error when trying to connect to redshift db

### ISSUE THREAD

**Issue by jcox92** [dbcli/pgcli#282]
After installing pgcli using brew, I get the following error when I try to connect to a redshift DB:

```
Traceback (most recent call last):
  File "/usr/local/Cellar/pgcli/0.17.0/libexec/bin/pgcli", line 9, in <module>
    load_entry_point('pgcli==0.17.0', 'console_scripts', 'pgcli')()
  File "/usr/local/lib/python2.7/site-packages/click-4.0-py2.7.egg/click/core.py", line 664, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click-4.0-py2.7.egg/click/core.py", line 644, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/site-packages/click-4.0-py2.7.egg/click/core.py", line 837, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/site-packages/click-4.0-py2.7.egg/click/core.py", line 464, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/Cellar/pgcli/0.17.0/libexec/lib/python2.7/site-packages/pgcli/main.py", line 416, in cli
    pgcli.run_cli()
  File "/usr/local/Cellar/pgcli/0.17.0/libexec/lib/python2.7/site-packages/pgcli/main.py", line 197, in run_cli
    self.refresh_completions()
  File "/usr/local/Cellar/pgcli/0.17.0/libexec/lib/python2.7/site-packages/pgcli/main.py", line 351, in refresh_completions
    completer.set_search_path(pgexecute.search_path())
  File "/usr/local/Cellar/pgcli/0.17.0/libexec/lib/python2.7/site-packages/pgcli/pgexecute.py", line 244, in search_path
    cur.execute(self.search_path_query)
psycopg2.ProgrammingError: function unnest(name[]) does not exist
HINT:  No function matches the given name and argument types. You may need to add explicit type casts.
```

Any idea what might be causing this?

**Comment by amjith** [comment]
The issue is because pgcli is targeting Postgres 9.x and above. 

There is effort underway to try and support older versions. Please see https://github.com/dbcli/pgcli/issues/18 for details.

**Comment by jcox92** [comment]
Ah very good! Thanks for the quick response!

**Comment by ixtli** [comment]
I also have this issue, but am satisfied with the response :) (Also I would prefer to see a more stable and featureful client instead of entering the morass of supporting older versions of postgres :) )

**Comment by dannyeuu** [comment]
There are something to do with redshift? support or a new CLI for that? RedShift is being each time more usefull, we migrate some PG do RedShift here in my enviroment.

**Comment by amjith** [comment]
This is now available in v1.0. Please upgrade. 

```
pip install -U pgcli
```
